Model/Order/PlaceOrder.php

```diff
@@ -465,74 +343,23 @@ private function getValidationErrorResponse(string $message): ResultInterface
         );
     }
 
-    /**
-     * @return array{
-     *      data?: array{
-     *          total: int,
-     *          transactions: array{
-     *              gateway: string,
-     *              payment_id: string,
-     *              amount: int,
-     *              transaction_id: string,
-     *              currency: string,
-     *              step: string,
-     *              status: 'success'|'failed'|'',
-     *              tender_type: string,
-     *              tender_details: array{
-     *                  brand: string,
-     *                  last_four: string,
-     *                  bin: string,
-     *                  expiration: string
-     *              },
-     *              gateway_response_data: string[]
-     *          }[]
-     *      },
-     *      errors?: array{
-     *          message: string,
-     *          type: string,
-     *          field: string,
-     *          severity: string,
-     *          sub_type: string,
-     *          code?: string,
-     *          transactions?: array{
-     *              gateway: string,
-     *              payment_id: string,
-     *              amount: int,
-     *              transaction_id: string,
-     *              currency: string,
-     *              step: string,
-     *              status: 'success'|'failed'|'',
-     *              tender_type: string,
-     *              tender_details: array{
-     *                  brand: string,
-     *                  last_four: string,
-     *                  bin: string,
-     *                  expiration: string
-     *              },
-     *              gateway_response_data: string[]
-     *          }[]
-     *      }[]
-     *  }
-     * @throws Exception
-     */
     private function getAuthorizedPayments(string $publicOrderId, int $websiteId): array
     {
         $url = sprintf('checkout/orders/{{shopId}}/%s/payments/auth/full', $publicOrderId);
         $result = $this->client->post($websiteId, $url, []);
         $errors = $result->getErrors();
-
-        if (array_key_exists('errors', $errors) && count($errors) > 0) {
```

Comment:
There was a bug in this line that was not caught during the initial development. It should read like this: `if (array_key_exists('errors', $errors) && count($errors['errors']) > 0) {` (note that the second part of the condition should check the length of the inner `errors` array instead of the outer array. If I recall correctly, during initial testing, I was seeing in some instances that rather than returning the errors in the expected and documented format, I got an array of error messages instead. This conditional was intended to handle that edge case. Unfortunately, I neglected to add a test case for that to the integration test.